package.json

```diff
@@ -170,6 +170,6 @@
   ],
   "license": "MIT",
   "engines": {
-    "node": "^12.22.0 || ^14.17.0 || >=16.0.0"
+    "node": "^18.18.0 || ^20.9.0 || >=21.1.0"
```

Comment:
Hey sorry for the random comment but why `^18.18.0` and not `^18.0.0`? I've been trying to dig around for this and can't find the reason. I've got the node engine set to `>=18` in my app so I'd need to upgrade which is fine, I'm just curious :)

Reply:
It's the first LTS release for 18

Reply:
have you went through the discussion in the related [issue](https://github.com/eslint/eslint/issues/17595)?

Reply:
> have you gone through the discussion in the related [issue](https://github.com/eslint/eslint/issues/17595)?

Yeah but I'm missing why those versions were chosen. The only thing I see is it being the latest versions at the time.